File: Java/UberX.java
class UberX extends Car
{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model)
    {
        super(license, driver); //Asi se acceden a las propiedades de la superclase
        this.brand = brand;
        this.model = model;        
    }    
}

File: PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car
{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model) 
    {
        parent::__construct($license, $driver);  //Equivale a llamar la superclase 
        $this->brand = $brand;
        $this->model = $model;
    }
}
?>